aebn_dl.py

```diff
@@ -264,6 +262,8 @@ def _download_segments(self):
                 sys.exit(f"Scene {self.scene_n} not found!")
 
         for stream_type in self.stream_types:
+            stream_id = ""
+            tqdm_desc = ""
```

Comment:
do we really need them here?
seems fine as it is

Reply:
It is fine as is, but Pylance complains that the variable is possibly unbound. I could leave it as it was, but I like to have error and bug free code :)